config.ini

```diff
@@ -18,7 +18,10 @@ AcPosition=1
 NumberOfInverters=1
 
 # Which DTU to be used ahoy, opendtu or template
-DTU=opendtu
+DTU=ahoy
+
+# send YieldDay instead of YieldTotal
```

Comment:
please add a comment, that this can break the Reporting in VRM if changed to 1 and change to 0 as default.

Reply:
Why do you think this would break the statistics in VRM if `useYieldDay` is set to 1? In my case, the problem arose from `YieldTotal` being used. 
I believe using `YieldDay` is the safer way in any case.

Reply:
as it changes the default for people already using the script, as we introduce this newly, I would not want to change the default behaviour. I agree with you, it might be safer if you start now.

Reply:
Oh yes. That's correct (at least most probably).